add node-red terminology page (#2121)
FlowFuse/website: src/node-red/terminology/index.md

---
eleventyNavigation:

>   key: Terminology

Reviewer: Is the `parent` added here magically? Or should that explicitly still be set? I *do* see the correct nav rendered, I just don't understand how it works. :)

Reviewer 2: The `/node-red/node-red.json` adds the `learningResources` tag to all the content inside `/node-red/`, so they're all part of the same collection. If you add a key, the plugin will add it at the first level of the navigation of that collection, unless you assign a parent, then it would be nested under that parent in the navigation structure.

meta:
   title: Node-RED Terminology
   description: Explore key terms and concepts used within the Node-RED community, including flows, nodes, subflows, workspace, and more.
   keywords: node-red, node-red terminology, node-red concepts, node-red key terms
---

## Node-RED Terminology

> This documentation is designed to help you understand the key terms and concepts used within the Node-RED community. Having a standardized vocabulary ensures clarity and consistency in our communications.

Reviewer 3: ```suggestion
Here are the key terms and concepts used within the Node-RED community to ensure clarity and consistency when communicating across projects. 
```

---
eleventyNavigation:
  key: Terminology

> title: Node-RED Terminology
> description: Explore key terms and concepts used within the Node-RED community, including flows, nodes, subflows, workspace, and more.
> tags:
>     - node-red
>     - node-red terminology
>     - node-red concepts
>     - node-red key terms

Reviewer 2: ```suggestion
meta:
   title: Node-RED Terminology
   description: Explore key terms and concepts used within the Node-RED community, including flows, nodes, subflows, workspace, and more.
   keywords: node-red, node-red terminology, node-red concepts, node-red key terms
```

Hi @sumitshinde-84, the title is being rendered in a section that we don't need for this. This one is on me, I should change the template, I tested a couple just now but none seems like the right fit. I'll take care of that in a different PR. However, we can still have a title without rendering it, if we add it under the `meta` key. 

Regarding the tags, Eleventy uses them to build posts collections, so this would be added to `/blog/node-red/`, which we don't want. So for the content that is not a blog article, let's add `meta.keywords` instead. [Here's a short explanation](https://flowfuse.com/handbook/customer/marketing/website/#meta-keywords) on how this works.

Author: Thanks @Yndira-E for the update
